frontends/user-articles/UserArticlesSidebar.tsx

```diff
@@ -58,6 +61,18 @@ export const UserArticlesSidebar: FC = () => {
     if (isPublishPressed) togglePublishModal();
   }, [isSavePressed, isPublishPressed]);
 
+  const StatusIcon = useMemo(() => {
```

Comment:
<img width="394" alt="Screenshot 2023-08-13 at 11 58 30" src="https://github.com/Juratbek/upper_frontend/assets/55176773/6ebd6c7a-b11d-4ae4-aafa-ed1b15fcb1e3">

status icon is not centered vertically

Reply:
Done